modules/vpc-endpoints/main.tf

```diff
@@ -44,7 +44,11 @@ resource "aws_vpc_endpoint" "this" {
     }
   }
 
-  tags = merge(var.tags, try(each.value.tags, {}))
+  tags = merge(
+    var.tags,
+    { "Name" = "${each.key}-vpc-endpoint" },
```

Comment:
```suggestion
    { "Name" = replace(try(each.value.service_endpoint, data.aws_vpc_endpoint_service.this[each.key].service_name, ""), ".", "-"") },
```

Reply:
Thanks @bryantbiggs for quick suggestions, I agree that it's better approach to use service name.

Reply:
meh, I deployed that and forgot the `com.amazonaws.<region>` is in there so reverted back to your original suggestion

Reply:
At the beginning I was also adding suffix `-vpc-endpoint` . Can I restore it too ?